src/components/Tool/Appeal/AddAppealForm.tsx

```diff
@@ -97,13 +98,31 @@ const AddAppealForm = (): ReactElement => {
   const { t } = useTranslation();
   const accountListId = useAccountListId() || '';
   const { enqueueSnackbar } = useSnackbar();
-  const { data, loading } = useGetContactTagsQuery({
+  const {
+    data: contactFilterTags,
+    loading: loadingTags,
+  } = useGetContactTagsQuery({
     variables: { accountListId },
   });
+  const {
+    data: contactFilterGroups,
+    loading: loadingStatuses,
+  } = useContactFiltersQuery({
+    variables: {
+      accountListId,
+    },
+  });
+
+  const contactStatuses = contactFilterGroups?.accountList.contactFilterGroups
+    ? (contactFilterGroups.accountList.contactFilterGroups.filter(
+        (group) => group.name === 'Status',
+      )[0].filters[0] as MultiselectFilter).options
```

Comment:
If you only want one, you can use `find`:
```suggestion
    ? (contactFilterGroups.accountList.contactFilterGroups.find(
        (group) => group.name === 'Status',
      ).filters[0] as MultiselectFilter).options
```
Might be good to find the status filter by `filterKey === 'status'` instead of index, just in case the API response changes in the future.